server/routerlicious/packages/services-core/src/runWithRetry.ts

```diff
@@ -7,6 +7,19 @@ import { delay } from "@fluidframework/common-utils";
 import { Lumber, LumberEventName, Lumberjack } from "@fluidframework/server-services-telemetry";
 import { NetworkError } from "@fluidframework/server-services-client";
 
+interface runWithRetryParams<T> {
```

Comment:
nit: most (if not all) interfaces in the FF repo are UpperCamelCase with an "I" prefix. So this would be `IRunWithRetryParams<T>`